### PR TITLE
New Extension: iHosts

### PR DESCRIPTION
**Pull request by 404J**
## Description

Make hosts management more efficient and easier~

## Screencast

ihosts-1
ihosts-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by 404J** [comment]
@sxn hi👋，It's time for this PR to move forward😁

File: extensions/ihosts/src/utils/file.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+export function getContentFromFile(path: string): string {
+  return fs.readFileSync(path, "utf8");
+}
+
+export async function writeSysHostFile(content: string) {
+  const tempHostsPath = path.join(os.tmpdir(), TempSystemHostFileName);
+  fs.writeFileSync(tempHostsPath, content, "utf8");
+  checkSysHostAccess() || (await getSysHostAccess());
+  await runAppleScript(`do shell script "cat ${tempHostsPath} > ${SystemHostFilePath} && rm -rf ${tempHostsPath}"`);
```
Comment (mathieudutour) [inline extensions/ihosts/src/utils/file.ts:26]: any reason to use an applescript instead of a child process directly?

Comment (404J) [reply extensions/ihosts/src/utils/file.ts]: Indeed, there is no need to use apple script～ I have pushed a commit to make the change

**Review by mathieudutour — APPROVED** [review]

**Review by mathieudutour — APPROVED** [review]
Looks good!

**Pull request merged**